cfme/tests/automate/test_simulation.py

```diff
@@ -86,3 +90,128 @@ def test_assert_failed_substitution(copy_class):
             request="Call_Instance",
             execute_methods=True,
         )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(automates=[1445089])
+def test_automate_simulation_result_has_hash_data(custom_instance):
+    """
+    The UI should display the result objects if the Simulation Result has
+    hash data.
+
+    Bugzilla:
+        1445089
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: automate
+        testSteps:
+            1. Create a Instance under /System/Request called ListUser, update it so that it points
+               to a ListUser Method
+            2. Create ListUser Method under /System/Request, paste the Attached Method
+            3. Run Simulation
+        expectedResults:
+            1.
+            2.
+            3. The UI should display the result objects
+    """
+    script = dedent(
```

Comment:
@ganeshhubale I think this script should be stored in `data/ruby_scripts/` and then imported. You could even just make it a python file with the same string. But it'd be good to keep all these scripts in a centralized location. What do you think?

Reply:
Hey @john-dupuy 
I agree but instead of creating files under ```data/ruby_scripts/```; I have added ruby scripts in already available file ```tests/automate/__init__.py```

Reply:
@ganeshhubale Thanks, that works too!